config/vufind/Blender.ini

```diff
@@ -234,6 +234,10 @@ exclude = blender_backend
 ; Do we want any facets to be collapsed by default?
 ;collapsedFacets = *
 
+; Enable JS feature to select multiple facets without reloading the result page
+; default : false (behaviour disabled)
+multiFacetsSelection = true
```

Comment:
If the default is false, should we comment out this setting by default? It's a bit confusing to have the pre-configured value be out of sync with the default. If you think this should be on by default everywhere, maybe we should consider changing the default behavior accordingly. If you're not sure of the best option, it might be worth having a community discussion to see if there are strong preferences one way or another before you go to the trouble of editing a bunch of files. :-) I could put this on a Community Call agenda, or we could try reaching out via Slack and/or the mailing lists to collect some opinions.

Reply:
I made the modifications, it makes more sense
I let you decide if we should keep this review open

Reply:
Did you have time to discuss this point?

Reply:
I'm still waiting for further feedback from @sturkel89 and @crhallberg. Hopefully we'll hear more next week when @sturkel89 returns from vacation. :-)

Reply:
Any thoughts on this one?

Reply:
Not sure yet -- I'll check in with @sturkel89 and @crhallberg for their opinions; both are back in the office this week.